Makefile

```diff
@@ -23,6 +23,8 @@ build_readme:
 	cat docs/readme/get_started_config.md >> ${README_FILE}
 	echo "\n## Get started using Python\n" >> ${README_FILE}
 	cat docs/readme/python_examples.md >> ${README_FILE}
+	echo "\n## Usage with PyTorch Metric Learning\n" >> ${README_FILE}
```

Comment:
Why don't we just use `cat *.md > ${README_FILE}`? We can add headers in the individual files and name them in way way the should be arranged: `0_header.md`, `1_faq.md` and so on.